### DISCUSSION TITLE
Unable to create same erp-express app name on ui

### DISCUSSION THREAD

**Discussion by huangyutongs** [argoproj/argo-cd#11037] [Q&A] [answered]
I now have an app-express application that needs to be deployed to 3 environments, dev, uat, pro, I can't create the same erp-express application name on the ui, how can I do best practice

**Comment by alexmt** [comment, marked as answer]
This is expected. There is no single application naming conventions. Typically environment is included in app name. E.g. `app-express-dev`, `app-express-uat`
